### PR TITLE
Upgrade cert manager

### PR DESCRIPTION
**Pull request by a-cool-train**
*Description of changes:*
- Upgrade Cert manager version
- CAPI uses the latest version of cert-manager: https://github.com/kubernetes-sigs/cluster-api/blob/d2f6ae112017d91caac80b3cddff0c2babcc4069/cmd/clusterctl/client/config/cert_manager_client.go#L32
- Restructured the patches and make sure they work with the latest version

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by TerryHowe** [comment]
/retest

Ephemeral Athena problem maybe?

**Comment by a-cool-train** [comment]
This PR depends on this issue getting fixed upstream: https://github.com/cert-manager/cert-manager/issues/5750

**Comment by g-gaston** [comment]
Although it's true capi uses 1.11, we haven't upgraded capi yet. So I'm not sure we bump this until we do that. Unless there is a way to do it but keep using 1.9 when installing for capi.

**Review by TerryHowe — APPROVED** [review]
/lgtm
/approve

File: projects/cert-manager/cert-manager/manifests/cert-manager.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Copyright 2021 The cert-manager Authors.
```
Comment (abhay-krishna) [inline projects/cert-manager/cert-manager/manifests/cert-manager.yaml:1]: For posterity: 
```bash
➜  ~ curl -sL https://github.com/cert-manager/cert-manager/releases/download/v1.9.1/cert-manager.yaml | wc -l
    5422
➜  ~ curl -sL https://github.com/cert-manager/cert-manager/releases/download/v1.11.0/cert-manager.yaml | wc -l                  
    5529
```
There are only about 107 lines added to the manifest between 1.9.1 and 1.11.0, but they moved around the CRDs and fields A LOT so the diff looks huge.

File: projects/cert-manager/cert-manager/manifests/cert-manager.yaml [hunk 2]
```diff
@@ -5233,25 +5322,32 @@ spec:
...
+          image: "quay.io/jetstack/cert-manager-controller:v1.11.0"
           imagePullPolicy: IfNotPresent
           args:
-            - --v=2
-            - --cluster-resource-namespace=$(POD_NAMESPACE)
-            - --leader-election-namespace=kube-system
+          - --v=2
+          - --cluster-resource-namespace=$(POD_NAMESPACE)
+          - --leader-election-namespace=kube-system
+          - --acme-http01-solver-image=quay.io/jetstack/cert-manager-acmesolver:v1.11.0
```
Comment (abhay-krishna) [inline projects/cert-manager/cert-manager/manifests/cert-manager.yaml:5335]: This manifest introduces a new image parameter that needs to be replaced in the `build/create_manifest.sh` script.

**Comment by a-cool-train** [comment]
This PR upgrades cert-manager: https://github.com/aws/eks-anywhere-build-tooling/pull/1726

**Pull request closed without merging**